spec/rails_helper.rb

```diff
@@ -31,6 +31,9 @@
 require Rails.root.join('spec/support/factory_bot')
 require Rails.root.join('spec/support/devise')
 
+Capybara.javascript_driver = :selenium_chrome_headless
+Capybara.ignore_hidden_elements = false
```

Comment:
From the README (see 5f835f94d1e5d9e873c4d4743ed4ec3ca4d6797e), I think that this change is still correct and should stay in (with chrome, because it works in CI)

Reply:
I'm not sure I undestand this correct:
Do we use Firefox now or is ist still Chrome? Does this line mean, that tests with the js: true tag use the Capybara.javascript_driver?

Reply:
As stated in the commit, tests with `js: true` use the `javascript_driver` whereas the other driver should be left to the default one for test performance. We used firefox for a short period in this PR, but it is changed back to chrome (refer to the readme of capybara for more info)